### PR TITLE
Hash Table Lab

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code

### CODE REVIEW

File: challenges/src/main/java/Java/hashtable/Hashtable.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        Node node = new Node(key, value);
+        if (hashbuckets[hashedKey] == null){
+            hashbuckets[hashedKey] = node;
+        } else {
+            node.setNext(hashbuckets[hashedKey]);
+            hashbuckets[hashedKey] = node;
+        }
+    }
+
+    public Object get(String key){
```
Comment (evanslaton) [inline challenges/src/main/java/Java/hashtable/Hashtable.java:29]: Where you not able to return `T`? By returning an Object user's will have to cast the returned value into the data type they're trying to use.

File: challenges/src/main/java/Java/hashtable/Node.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package Java.hashtable;
+
+class Node<T> {
+    public String key;
```
Comment (evanslaton) [inline challenges/src/main/java/Java/hashtable/Node.java:4]: If you have getters/setters you want to make these `private` instead of `public`
